app/models/media.rb

```diff
@@ -50,7 +50,7 @@ def doi_id=(value)
 
   def set_defaults
     current_media =
-      Media.where(dataset: dataset).order("media.created DESC").first
+      Media.where(dataset: dataset).order("created DESC").first
```

Comment:
Specifying a table name is only necessary on complex queries, simple queries like this one you can just specify the column as there is no ambiguity as to where the column belongs